src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -920,6 +931,11 @@ private void ParseHelper(string[] args)
                     _sshServerMode = true;
                     ParametersUsed |= ParameterBitmap.SSHServerMode;
                 }
+                else if (MatchSwitch(switchKey, "noprofileloadtime", "noprofileloadtime"))
```

Comment:
Should we consider having `-nplt`?

Reply:
I like that.  But how about `-noplt` so the alias is a bit more consistent with the other `-no` aliases: `-nol, -nop, -noni`?

Reply:
Nevermind, when I try that I run smack into this dbg assert:
```
Dbg.Assert(match.Contains(smallestUnambiguousMatch), "sUM should be a substring of match");
```